docs/generated/reference/openapi/openapi_generated.go

```diff
@@ -0,0 +1,3257 @@
+// +build !ignore_autogenerated
```

Comment:
This file should not be committed anymore - recent changes to docs generation mean we no longer generate this .go file 😄